<sdf version='1.10' xmlns:ros2_control="http://playerstage.sourceforge.net/gazebo/xmlschema/#ros2_control">
      
  <world name='fito_world'>

    <physics name='1ms' type='ode'>
      <max_step_size>0.0050000000000000001</max_step_size>
      <real_time_factor>1</real_time_factor>
      <real_time_update_rate>1000</real_time_update_rate>
    </physics>
<plugin
  filename="gz-sim-physics-system"
  name="gz::sim::systems::Physics">
</plugin>
<plugin
  filename="gz-sim-user-commands-system"
  name="gz::sim::systems::UserCommands">
</plugin>
<plugin
  filename="gz-sim-scene-broadcaster-system"
  name="gz::sim::systems::SceneBroadcaster">
</plugin>
<plugin
  filename="gz-sim-sensors-system"
  name="gz::sim::systems::Sensors">
  <render_engine>ogre2</render_engine>
</plugin>
<plugin
  filename="gz-sim-imu-system"
  name="gz::sim::systems::Imu">
</plugin>
    <gravity>0 0 -9.8000000000000007</gravity>
    <magnetic_field>5.5644999999999998e-06 2.2875799999999999e-05 -4.2388400000000002e-05</magnetic_field>
    <atmosphere type='adiabatic'/>
    <scene>
      <ambient>0.400000006 0.400000006 0.400000006 1</ambient>
      <background>0.699999988 0.699999988 0.699999988 1</background>
      <shadows>true</shadows>
    </scene>
    <model name='bush'>
      <pose>0 0 0 0 0 0</pose>
      <link name='link'>
        <visual name='bush_visual'>
          <pose>0 0 0 0 0 0</pose>
          <geometry>
            <mesh>
              <uri>model://my_robot/meshes/bushes.STL</uri>
              <scale>0.0003 0.0003 0.0003</scale>
            </mesh>
          </geometry>
          <material>
            <ambient>0.100000001 0.5 0.100000001 1</ambient>
            <diffuse>0.300000012 0.699999988 0.300000012 1</diffuse>
            <specular>0.100000001 0.100000001 0.100000001 1</specular>
          </material>
        </visual>
        <pose>0 0 0 0 0 0</pose>
     
        <enable_wind>false</enable_wind>
      </link>
      <static>true</static>
      <self_collide>false</self_collide>
    </model>
   

  
    


  <model name="ground_plane">
    <static>true</static>
    <link name="link">
      <collision name="collision">
        <geometry>
          <plane>
            <normal>0 0 1</normal>
            <size>100 100</size>
          </plane>
        </geometry>
        <surface>
          <friction>
            <ode>
              <mu>100</mu>
              <mu2>50</mu2>
            </ode>
          </friction>
        </surface>
      </collision>
      <visual name="visual">
        <geometry>
          <plane>
            <normal>0 0 1</normal>
            <size>100 100</size>
          </plane>
        </geometry>
        <material>
          <ambient>0.4 0.3 0.1 1</ambient>
          <diffuse>0.5 0.4 0.2 1</diffuse>
          <specular>0.1 0.1 0.1 1</specular>
        </material>
      </visual>
    </link>
  </model>


    <light name='sun' type='directional'>
      <pose>0 0 10 0 0 0</pose>
      <cast_shadows>true</cast_shadows>
      <intensity>1</intensity>
      <direction>-0.5 0.10000000000000001 -0.90000000000000002</direction>
      <diffuse>0.800000012 0.800000012 0.800000012 1</diffuse>
      <specular>0.200000003 0.200000003 0.200000003 1</specular>
      <attenuation>
        <range>1000</range>
        <linear>0.01</linear>
        <constant>0.90000000000000002</constant>
        <quadratic>0.001</quadratic>
      </attenuation>
      <spot>
        <inner_angle>0</inner_angle>
        <outer_angle>0</outer_angle>
        <falloff>0</falloff>
      </spot>
    </light>
  </world>
</sdf>
